### ISSUE TITLE
summary4

### ISSUE THREAD

**Issue by shobhit-v** [shobhit-v/demo#85]
## Issue Description
Impact: Critical
Summary: summary4Description: The text alternative is missing.
Standards: wcag2a, wcag2aaProduct: CustomView-TestRunsTest Case Name: Customer_View_Testrun_TC_195952Test Run Name: Customer_View_Testrun_TC_195952_08-12-21-08:19Checkpoint Group: 1.1 Text AlternativesIssue Type: AccessibilityUnit Type: PageTest Unit: Customer_View_Testrun_PageScreenshots: Recommended to fix: Digital Asset Type: Native Mobile iOSRelease: REL4Environment: ENV4Platform: PLFM4Assistive technology: AT4Source Code: Checkpoint: Alternative Text (Active Images) (1.1.1.a)Method: ManualTested On URL: http://abcdcomputech.dequecloud.com/More Info: Issue Comments:
